### PR TITLE
DevWs for SHINA SYSTEM containing containing SiHas Multipurpose Sensor

### PR DESCRIPTION
**Pull request by shinasys**
1. Report time adjustment.
2. Motion uses a OCCUPANCY_SENSING_CLUSTER, OCCUPANCY_SENSING_OCCUPANCY_ATTRIBUTE. So, delete unnecessary parts.

### CODE REVIEW

File: devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy [hunk 1]
```diff
@@ -132,9 +132,7 @@ private Map translateZoneStatus(ZoneStatus zs) {
     // Some sensor models that use this DTH use alarm1 and some use alarm2 to signify motion
     if (isDSM300()) {
     	return (zs.isAlarm1Set() || zs.isAlarm2Set()) ? getContactResult('open') : getContactResult('closed')
-    } else {    
-    	return (zs.isAlarm1Set() || zs.isAlarm2Set()) ? getMotionResult('active') : getMotionResult('inactive')
-    }
```
Comment (KKlimczukS) [inline devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy:137]: Was it intentional ?
If You leave it like this then this function will not return anything for models other than **isDSM300**

File: devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy [hunk 2]
```diff
@@ -286,19 +284,19 @@ def configure() {
...
-        configCmds += zigbee.configureReporting(ILLUMINANCE_MEASUREMENT_CLUSTER, ILLUMINANCE_MEASUREMENT_MEASURED_VALUE_ATTRIBUTE, DataType.UINT16, 20, 3600, 10/*10 lux*/)
+        configCmds += zigbee.configureReporting(ILLUMINANCE_MEASUREMENT_CLUSTER, ILLUMINANCE_MEASUREMENT_MEASURED_VALUE_ATTRIBUTE, DataType.UINT16, 15, 3600, 1/*1 lux*/)
     }
 
     if (isUSM300() || isOSM300()) {
         configCmds += zigbee.configureReporting(OCCUPANCY_SENSING_CLUSTER, OCCUPANCY_SENSING_OCCUPANCY_ATTRIBUTE, DataType.BITMAP8, 1, 600, 1)
     }
 
-    if (isDSM300()) {
+    if (isDSM300() || isUSM300() || isOSM300()) {
```
Comment (KKlimczukS) [inline devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy:299]: double-checking ->  USM300() and OSM300 dont use IAS_ZONE_CLUSTER/ATTRIBUTE_IAS_ZONE_STATUS right ?

File: devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy (hunk 1 above)
Comment (shinasys) [reply devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy:137]: @KKlimczukS Yes. Other models is unnecessary code.

File: devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy (hunk 2 above)
Comment (shinasys) [reply devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy:299]: yes.

**Review by KKlimczukS — APPROVED** [review]

**Pull request merged**